apps/portals/src/configurations/elportal/synapseConfigs/studies.ts

```diff
@@ -44,7 +47,8 @@ export const studyCardConfiguration: CardConfiguration = {
       'species',
       'specimenType',
       'program',
-      'grant',
+      'projectShortName',
```

Comment:
This should probably be the project full name, but there's an existing data join issue. Let's go with this for now and fix later.